aten/src/ATen/core/typeid.h

```diff
@@ -134,253 +237,118 @@ struct TypeNameRegisterer {
  */
 class AT_CORE_API TypeMeta {
```

Comment:
Can we now (in a later PR) rename TypeMeta to `DataType`? :)

Reply:
All my arguments from before still apply, also the argument that I don't have a strong opinion and you can rename it if you want ;)